### DISCUSSION TITLE
Transparency should be the default secondary color

### DISCUSSION THREAD

**Discussion by JayKayAce** [ni/labview-icon-editor#57] [Ideas]
# Problem Statement
As an icon developer, I would like to be able to permanently select the transparent color so that I can use the secondary color as an eraser. The implementation resets any transparent color to an off-white color when you leave the icon editor. This should not be the default behavior.

**Comment by svelderrainruiz** [comment]
In my opinion this idea makes sense, unless there are any objections, i vote we move this to our next planned feature.
